Shopipy.BusinessManagement/Mappings/BusinessMappingProfile.cs

```diff
@@ -0,0 +1,35 @@
+using AutoMapper;
+using Shopipy.BusinessManagement.DTOs;
+using Shopipy.Persistence.Models;
+
+namespace Shopipy.BusinessManagement.Mappings;
+
+public class BusinessMappingProfile : Profile
+{
+    public BusinessMappingProfile()
+    {
+        // Map Business -> BusinessResponseDto
+        CreateMap<Business, BusinessResponseDto>()
+            .ForMember(dest => dest.BusinessId, opt => opt.MapFrom(src => src.BusinessId))
+            .ForMember(dest => dest.Name, opt => opt.MapFrom(src => src.Name))
+            .ForMember(dest => dest.Address, opt => opt.MapFrom(src => src.Address))
+            .ForMember(dest => dest.Phone, opt => opt.MapFrom(src => src.Phone))
+            .ForMember(dest => dest.Email, opt => opt.MapFrom(src => src.Email))
+            .ForMember(dest => dest.VatNumber, opt => opt.MapFrom(src => src.VatNumber))
+            .ForMember(dest => dest.BusinessType, opt => opt.MapFrom(src => src.BusinessType))
+            .ForMember(dest => dest.CreatedAt, opt => opt.MapFrom(src => src.CreatedAt))
+            .ForMember(dest => dest.UpdatedAt, opt => opt.MapFrom(src => src.UpdatedAt));
```

Comment:
The point of automapper is less work. If property names are the same automapper will do the mapping automatically. @ashkema

Reply:
Feel free to resolve this comment afterwards.